src/views/pages/globalping/users.html

```diff
@@ -0,0 +1,448 @@
+<link rel="ractive" href="../../r-page-globalping.html" name="r-page">
+<link rel="ractive" href="../../components/gp-header.html" name="c-gp-header">
+<link rel="ractive" href="../../components/gp-footer.html" name="c-gp-footer">
+<link rel="ractive" href="../../components/notification.html" name="c-notification">
+
+<r-page noYield="{{noYield}}" title="{{title}}" description="{{description}}">
+	<c-notification></c-notification>
+	<c-gp-header additionalClasses="header-with-gp-translucent-bg"></c-gp-header>
+
+	{{#if !@shared.isServer && !@shared.googleMapsLoaded && username}}
+		<script>
+			function onDocumentReady (fn) {
+				if (document.readyState !== 'loading') {
+					fn();
+				} else {
+					document.addEventListener('DOMContentLoaded', fn);
+				}
+			}
+			/* eslint-disable */
+			function initMap () {
+				onDocumentReady(function () {
+					app.loaded = true;
+
+					if (app.router.route && app.router.route.view && app.router.route.view.initMap) {
+						app.router.route.view.initMap();
+					}
+				});
+			}
+		</script>
+		<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBTICLY5LgWNJ0Uly61Q1sNXz_1KZIECwg&amp;language=en&amp;callback=initMap&libraries=marker&loading=async" async></script>
+		<script src="https://cdn.jsdelivr.net/npm/@googlemaps/markerclusterer@2.5.3/dist/index.umd.min.js"></script>
+	{{/if}}
+
+	<div class="p-users">
+		<div class="please-wait-sign {{#if username.length}} not-displayed{{/if}}">
+			Please wait while user data loads.
+		</div>
+
+		<div class="page-content {{#unless username.length}} not-displayed{{/unless}}">
+			<div class="page-content_side-menu">
+				<div class="menu-section">
+					<div class="section-list">
```

Comment:
The user picture that was supposed to be here is actually just `https://github.com/${username}.png` so we can add it too.  Also, the username we're showing is the GitHub one. We can also add the number of probes and ASNs (just count the unique values across the user's probes). So that final version should be this:

![image](https://github.com/user-attachments/assets/06f8103c-affe-40a3-80dc-bebdb3b267c9)